devicemaps.json

```diff
@@ -1197,71 +395,60 @@
             }
           },
           {
-            "pciAddress": "0000:04:00.0",
+            "pciAddress": "0000:03:00.0",
             "type": "LAN",
             "speed": 1000,
             "name": "ge-3",
             "description": "Port 3 labeled on the device",
-            "bcpNetwork": {
-              "standaloneBranch": {
-                "name": "",
-                "description": ""
-              }
-            }
-          },
-          {
-            "pciAddress": "0000:03:00.1",
-            "type": "LAN",
-            "speed": 1000,
-            "name": "ge-4",
-            "description": "Port 4 labeled on the device",
-            "bcpNetwork": {
-              "standaloneBranch": {
-                "name": "",
-                "description": ""
-              }
-            }
-          },
-          {
-            "pciAddress": "0000:03:00.0",
-            "type": "LAN",
-            "speed": 1000,
-            "name": "ge-5",
-            "description": "Port 5 labeled on the device",
             "bcpNetwork": {
               "standaloneBranch": {
                 "name": "lan",
-                "description": "LAN network interface, connected to port 5 labeled on the device"
+                "description": "LAN network interface, connected to port 3 labeled on the device"
               }
             }
           },
           {
             "pciAddress": "0000:02:00.1",
             "type": "HAFabric",
             "speed": 1000,
-            "name": "ge-6",
-            "description": "Port 6 labeled on the device",
+            "name": "ge-4",
+            "description": "Port 4 labeled on the device",
             "bcpNetwork": {
               "standaloneBranch": {
                 "name": "ha-fabric",
-                "description": "HA fabric network interface, connected to port 6 labeled on the device"
+                "description": "HA fabric network interface, connected to port 4 labeled on the device"
               }
             }
           },
           {
             "pciAddress": "0000:02:00.0",
             "type": "HASync",
             "speed": 1000,
-            "name": "ge-7",
-            "description": "Port 7 labeled on the device",
+            "name": "ge-5",
+            "description": "Port 5 labeled on the device",
             "bcpNetwork": {
               "standaloneBranch": {
                 "name": "ha-sync",
-                "description": "HA sync network interface, connected to port 7 labeled on the device"
+                "description": "HA sync network interface, connected to port 5 labeled on the device"
               }
             }
           }
         ],
+        "lte": []
+      },
+      "SSR120-TAA": {
+        "alias": {
```

Comment:
do the 120 and 120-TAA use the same part_number and clei as well @BenMatase ? this doesn't seem right

Reply:
Just double checked in the old device-map, it uses the same clei and part_number. but not sure if that is intended or copy/pasta from before

Reply:
The `-TAA` are carbon copies of the original so this is correct. I can share the document/discussion that this was populated with.

Reply:
no need - just was quadruple checking 😀